CHANGELOG.rst

```diff
@@ -4,6 +4,8 @@ Changelog
 dev
 ---
 - Fix bug in plot_discrepancy for more than 6 parameters
+- Implement plot_gp for BayesianOptimization classes for plotting parameters vs. contours
+  in case when we have more than 2 parameters
```

Comment:
"parameters vs. contours" -> "discrepancies and pair-wise contours"

Should work with any number of parameters?

Reply:
I meant that this should work for cases of 1 and 2 parameters also. :D

Reply:
Well, I always check plottings from 1 up to 8-9 parameters if there is no discussion with parameters :)

Reply:
The reason why I wrote parameters number is because we had this problem. I will fix it with 'arbitrary' number of parameters (but not larger than 8).